.gitignore

```diff
@@ -4,3 +4,7 @@ bin
 
 ## Intermediate build files
 build
+
+
+## Swap Files
+*.swp
```

Comment:
Ah, thanks for the explanation. It's OK to add it to gitignore, but please change the comment to `Swap files for Vim`